src/array_operator.jl

```diff
@@ -52,18 +52,11 @@ Base.issymmetric(L::DiffEqArrayOperator) = L._issymmetric
 Base.ishermitian(L::DiffEqArrayOperator) = L._ishermitian
 Base.isposdef(L::DiffEqArrayOperator) = L._isposdef
 DiffEqBase.is_constant(L::DiffEqArrayOperator) = L.update_func == DEFAULT_UPDATE_FUNC
-function Base.expm(L::DiffEqArrayOperator)
-    tmp = full(L.A) # If not lazy then this is a no-op
-    tmp .*= L.α.coeff
-    out = expm(tmp)
-    if tmp === L.A
-        L.A ./= L.α.coeff # Undo change if not lazy
-    end
-    out
-end
+Base.full(L::DiffEqArrayOperator) = full(L.A) * L.α.coeff
```

Comment:
`.*` this so that way it's compatible with coefficient arrays. That'll come up in space-dependent coefficients and quasi-linear PDEs.